bookfinder/src/components/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { FaFacebook, FaInstagram, FaGithub } from "react-icons/fa";
+
+const Footer = () => {
+  return (
+    <div className="top-64 flex h-full w-full flex-col gap-10 bg-gray-300 p-10 xl:flex-row xl:gap-56">
```

Comment:
for medium sized screen, I see a gap on the right because you aligned them on the left as Jair mentioned also. I think for this size, it draws attention in not a good way.

Reply:
Okay, I get what you mean now. On medium size, Follow Us and Categories will come side by side, then Contact Us will come below them

Reply:
With my suggested code above, there of them will come side by side. I will try also when I have my laptop with me.

Reply:
Suggested changes have been implemented now. Please review again